domains/mohamad.json

```diff
@@ -4,8 +4,6 @@
         "email": "mj@naous.eu.org"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": ""[{"type":"CNAME","allowMultiple":false,"value":"mohamadev.github.io"}]""
```

Comment:
CNAME should just be a string, no fancy arrays needed.

Reply:
What? I used the dashboard to make that